README.md

```diff
@@ -1,225 +1,78 @@
-
-    * `height` *Number*
-
-        The height of the embedded signature request iFrame in pixels, when a specific container is specified. Minimum height allowed is `480`. Defaults to `900`.
-
-    * `healthCheckTimeoutMs` *Number*
-
-        An integer representing the number of milliseconds to wait for a response from the embedded signature request iFrame. If there is no response in that time, the iFrame will be closed automatically. `15000` is a recommended timeout. If this option is not specified, the embedded signature request will not close if idle.
-
-    * `hideHeader` *Boolean*
-
-        A boolean indicating whether or not to hide the signature request header. Only customers with embedded branding enabled are able to use this option. Defaults to `false`.
-
-    * `whiteLabelingOptions` *Object*
-
-        An object used to customize the signer experience for premium users with white-labeling capabilities.
-
-
-### close()
-
-**`hellosign.close()`**
-
-Closes the embedded signature request.
-
-
-## Demo
-
-There is a demo app available in this repository which is useful for examining how a simple HelloSign Embedded integration can work. To get it up and running, perform the following:
-
-1. Clone this repo.
-
-    ```
-    $ git clone git@github.com:HelloFax/hellosign-embedded.git
-    ```
-
-2. Run the setup script.
-
-    ```
-    $ npm run setup
-    ```
-
-3. Start the demo.
+<br/>
+<br/>
+<hr/>
 
-    ```
-    $ npm run demo
-    ```
+&copy; 2018 [HelloSign][external_hellosign]. All rights reserved.
 
-4. If all has gone well, you should now be able to access the demo at [http://localhost:3000](http://localhost:3000).
 
-In order to test out the demo, you'll need an API Key and Client ID for HelloSign's API. If you haven't done so yet, head on over to the [API settings for your account](https://www.hellosign.com/home/myAccount#api) on HelloSign and create an API App.
 
 
-## Other
 
-### TypeScript
 
-TypeScript bindings for HelloSign Embedded are available in the [DefinitelyTyped repo](https://github.com/DefinitelyTyped/DefinitelyTyped).
+[license]: ./LICENSE.md
 
+[badge_npm-version]: https://img.shields.io/npm/v/hellosign-embedded.svg
+[badge_npm-downloads]: https://img.shields.io/npm/dm/hellosign-embedded.svg
+[badge_david]: https://img.shields.io/david/hellosign/hellosign-embedded.svg
+[badge_travis]: https://img.shields.io/travis/hellosign/hellosign-embedded/master.svg
 
-<br/>
-<br/>
-<hr/>
+[wiki_home]: https://github.com/hellosign/hellosign-embedded/wiki
+[wiki_api-documentation]: https://github.com/hellosign/hellosign-embedded/wiki/API-Documentation-%28v2%29
+[wiki_changelog]: https://github.com/hellosign/hellosign-embedded/wiki/Changelog
+[wiki_migration-guide]: https://github.com/hellosign/hellosign-embedded/wiki/Migration-Guide-(v1-to-v2)
+[wiki_quickstart]: https://github.com/hellosign/hellosign-embedded/wiki/Quickstart
```

Comment:
These wiki links are broken right now, I'm assuming you'll bring the wiki back once v2 is released?

Reply:
These are broken because they link to @hellosign not @HelloFax -- We will transfer HelloSign Embedded to @hellosign on release, and then these links will work

Reply:
Cool, as long as they're on your radar.

Reply:
Yessireebob 👍

Reply:
The changelog however should be changed. I was going to make that a Wiki page but decided instead to keep it in a file

Reply:
Fixed